rdr_service/dao/genomics_dao.py

```diff
@@ -1408,6 +1445,14 @@ def get_feedback_equals_record_count(self):
 
         return list(results)
 
+    def get_feedback_record_counts_from_filepath(self, filepath):
+        with self.session() as session:
+            return session.query(
+                GenomicManifestFeedback.feedbackRecordCount
+            ).filter(
+                GenomicManifestFile.filePath == filepath
```

Comment:
We'll want to filter on `GenomicManifestFeedback.ignoreFlag != 1`. 
Also, you'll want to join from `GenomicManifestFeedback` to `GenomicManifestFile` on `GenomicManifestFile.id == GenomicManifestFeedback.inputManifestFileId`.